src/wp-admin/includes/class-walker-category-checklist.php

```diff
@@ -18,7 +18,10 @@
  */
 class Walker_Category_Checklist extends Walker {
 	public $tree_type = 'category';
-	public $db_fields = array ('parent' => 'parent', 'id' => 'term_id'); //TODO: decouple this
+	public $db_fields = array(
+		'parent' => 'parent',
+		'id'     => 'term_id',
+	); //TODO: decouple this
```

Comment:
not sure where the comment should go, but not on the )

Reply:
Does look odd but matches upstream:
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/includes/class-walker-category-checklist.php#L24

Reply:
Well, yes, they ran the same auto formatter. I guess it's a vague comment anyway.